src/web/services/course.service.ts

```diff
@@ -65,6 +65,14 @@ export class CourseService {
     return this.httpRequestService.put('/bin/course', paramMap);
   }
 
+  /**
+   * Restore a soft-deleted course by calling API.
+   */
+  restoreCourse(courseid: string): Observable<MessageOutput> {
```

Comment:
Just a question, should we change the http request on [line 246](https://github.com/TEAMMATES/teammates/blob/267d2d59eb0b6350ba4ecf76cf4d0583fb3481d6/src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.ts#L246) to use `courseService`? Probably ask for maintainer before doing so.

Reply:
At some point, yeah probably. Wrapping the change up with this PR sounds good, or we can always re-create it as a first timer issue.